ui/src/app/machines/views/MachineDetails/MachineNetwork/BondForm/types.ts

```diff
@@ -0,0 +1,29 @@
+import type { NetworkValues } from "../NetworkFields/NetworkFields";
+
+import type {
+  BondLacpRate,
+  BondMode,
+  BondXmitHashPolicy,
+} from "app/store/general/types";
+import type {
+  NetworkInterface,
+  NetworkInterfaceParams,
+} from "app/store/machine/types";
+
+export enum MIIOptions {
```

Comment:
Wouldn't `LinkMonitoring` be more accurate?

Reply:
Yep, good call.